viewpost.php

```diff
@@ -0,0 +1,20 @@
+<?php
+include ('config/init.php');
+$post = returnPostInformation($_REQUEST['postID']);
+siteHeader('blog * '.$post['title'].'','blog + inspiration');
+
+echo "
+<div class='postSubHeader'>
+<h2>$post[header]:</h2>
+</div>
+<div class='postBody'>
+$post[body]
+</div>
+<div class='backNextOnViewPostPage'>
+<table>
+<tr>
+<th><a href='Myblog.php'>back to blog + inspiration home</a></th>
+<th><a href='viewpost.php?postID=$post[nextPost]'>next post</a></th>
```

Comment:
This is pretty clever, and I think that it shows you have a good understanding of how these different pieces fit together (even if it seems confusing at times). As I said above, we should talk about other approaches, but I encourage you to keep adding little bonus functionality like this throughout the summer.